### PR TITLE
Added ios config, keystore for release

### PR DESCRIPTION
**Pull request by ManasMalla**

### CODE REVIEW

**Comment by ManasMalla** [comment]
1. Updated iOS config files to work with XCode 15
   Files affected
	• `ios/*`

2. Updated the names of variables in entities/models to avoid ambiguity
   Files affected
	• `/blog_model.dart`
	• `/event_model.dart`
	• `/user_model.dart`
	• `event.dart`
	• `post.dart`
	• `user.dart`

3. Added methods to fetch the current user from firebase auth
   Files affected
	• `data/repo/firebase_auth_repository_impl.dart`
	• `domain/repositories/firebase_auth_repository.dart`
	• `domain/usecases/firebase_auth_usecase.dart`

4. Added the `CreateProfileBloc` and its respective states and events
   Files affected
	• `main.dart`
	• `bloc/create_profile_module/*`
	• `injector.dart`

5. Reworked the splash workflow
   Files affected:
	• `splash_screen.dart - Implemented the `BlocConsumer` instead of `BlocBuilder`
	• `splash_bloc.dart`
	• `splash_state.dart`

6. Created the `CreateProfileBottomSheet` and respective widgets
   Files affected
	• `layouts/bottom_sheets/create_profile_bottom_sheet.dart`

**Review by sreetejadusi — APPROVED** [review]

**Comment by ManasMalla** [comment]
## Working on the create-profile workflow 

1. Added new methods to firestore use cases, repositories
	`queryData`: helps to query data in Firebase Firestore using the `equalsTo` key and value
	`queryDataCount`: helps to get the count of the queried rows in Firebase Firestore using the key, value and the `equalsTo` param.

2. UI Changes:
   • Need to add the username `TextField` and gender `SegmentedButton` to complete the `ModalBottomSheet` to create a new profile.
   • Handled the uniqueness of the username in the textfield

3. Added the `toJson` function for the user_model.dart

4. Added the `utils/either_utils.dart` that includes the `sequence` function that takes in an input of the type List<Either<A,B>> and returns Either<A, List<B>>

**Review by sampathbalivada — APPROVED** [review]
TLDR;

Approving the changes.
